forge/src/main/java/org/openjdk/skara/forge/gitlab/GitLabMergeRequest.java

```diff
@@ -82,13 +82,7 @@ public String id() {
 
     @Override
     public HostUser author() {
-        var username = json.get("author").get("username").asString();
-        var author = repository.forge().user(username);
-        if (author.isPresent()) {
-            return author.get();
-        } else {
-            throw new RuntimeException("Author of GitLab merge request unknown: " + username + "(maybe the user is inactive)");
-        }
+        return host.parseAuthorField(json);
```

Comment:
Did you test this? I would assume you need something like this:
```suggestion
        return host.parseAuthorField(json.get("author"));
```

Reply:
Yes, I tested it. We don't need to get author here. Because we get author in this method.

```
    HostUser parseAuthorField(JSONValue json) {
        return parseAuthorObject(json.get("author").asObject());
    }
```

Reply:
Oh, right, I misread the method name. Yes, that should work.